sdk/src/main/java/com/kin/ecosystem/transfer/view/AccountInfoActivity.java

```diff
@@ -17,6 +19,10 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        //On android above Oreo, you can not change orientation for Activity that has android:windowIsTranslucent=true
```

Comment:
@OrenZak  did you manage to look if orientation landscape looks fine for that design?